Steps/Steps.module.less

```diff
@@ -16,14 +16,14 @@
 			font-weight: @sand-steps-step-number-font-weight;
 		}
 
-		&.dots {
-			font-size: @sand-steps-step-dot-font-size;
+		&.highlightCurrent {
```

Comment:
Let's rename this to `highlightCurrentOnly` for clarity.

Reply:
It's fixed.